app/viewModels/locationOfGoods/LocationOfGoodsAnswersViewModel.scala

```diff
@@ -33,11 +34,16 @@ object LocationOfGoodsAnswersViewModel {
     def apply(userAnswers: UserAnswers, mode: Mode)(implicit messages: Messages, config: FrontendAppConfig): LocationOfGoodsAnswersViewModel = {
       val helper = new LocationOfGoodsCheckYourAnswersHelper(userAnswers, mode)
 
+      val inferredLocationType = userAnswers.get(ProcedureTypePage).flatMap {
+        case ProcedureType.Normal     => helper.locationType
+        case ProcedureType.Simplified => None
+      }
```

Comment:
Duplication of logic, but don't mind this as a quick fix. Might look to introduce an `InferredLocationTypePage` to be consistent with how we infer values elsewhere

Reply:
Yep I did notice that inferred pages. Didnt want to change too much in the cache for a quick fix. Will look to add something a bit better after e2e :)